test/ERC20RootVault.test.ts

```diff
@@ -0,0 +1,1149 @@
+                            ).toNumber() - 2;
+                        await ethers.provider.send("hardhat_setStorageAt", [
+                            this.vaultRegistry.address,
+                            "0x5", // address of nft index
+                            "0x0000000000000000000000000000000000000000000000000000000000000000",
+                        ]);
+                        await withSigner(
+                            this.erc20RootVaultGovernance.address, // what to write here????
+                            async (signer) => {
+                                await expect(
+                                    this.subject
+                                        .connect(signer)
+                                        .initialize(
+                                            this.nft,
+                                            [
+                                                this.usdc.address,
+                                                this.weth.address,
+                                            ],
+                                            randomAddress(),
+                                            [startNft]
+                                        )
+                                ).to.be.revertedWith(Exceptions.DUPLICATE);
+                            }
+                        );
+                    });
+                });
+
+                describe("when subvaultNft address is 0", () => {
+                    it(`reverts with ${Exceptions.ADDRESS_ZERO}`, async () => {
+                        const startNft = (
+                            await this.vaultRegistry.vaultsCount()
+                        ).toNumber();
+                        await ethers.provider.send("hardhat_setStorageAt", [
+                            this.vaultRegistry.address,
+                            "0x12", // address of vault
+                            "0x0000000000000000000000000000000000000000000000000000000000000000",
+                        ]);
+
+                        await withSigner(
+                            this.erc20VaultGovernance.address,
+                            async (signer) => {
+                                await expect(
+                                    this.subject
+                                        .connect(signer)
+                                        .initialize(
+                                            this.nft,
+                                            [
+                                                this.usdc.address,
+                                                this.weth.address,
+                                            ],
+                                            randomAddress(),
+                                            [startNft]
+                                        )
+                                ).to.be.revertedWith(
+                                    Exceptions.INVALID_INTERFACE // ADDRESS_ZERO
```

Comment:
Does not match with description above, rewrite the test pls. It is possible that description is wrong